src/components/SearchableSelect/SearchableSelect.tsx

```diff
@@ -350,6 +351,28 @@ class SearchableSelect extends React.Component<
 		onSearch(searchText, firstVisibleIndex);
 	};
 
+	handleExpand = ({
+		props,
+		event,
+	}: {
+		props: IDropMenuProps;
+		event: React.KeyboardEvent | React.MouseEvent;
+	}): void => {
+		const dropMenuProps = this.props.DropMenu;
+		dropMenuProps.onExpand && dropMenuProps.onExpand({ event, props: props });
+		this.setState({ isFocusOnSearchFieldRequired: true });
+	};
+
+	setSearchField = (e: SearchField) => {
+		if (e && this.state.isFocusOnSearchFieldRequired) {
+			this.setState({ isFocusOnSearchFieldRequired: false });
+			// use setTimeout to prevent scroll from safari
```

Comment:
Without setTimeout usage, safari is scrolling to top of rendered page when opening component from from bottom of page (detected from docspot rendered page)

Firefox and Chrome only require `{ preventScroll: true }` option parameter to avoid scrolling